lib/membrane_audio_mixer/live_queue.ex

```diff
@@ -0,0 +1,92 @@
+defmodule Membrane.AudioMixer.LiveQueue do
+  @moduledoc """
+  """
+  alias Membrane.RawAudio
+
+  def init(stream_format),
+    do: {:ok, %{queues: %{}, offsets: %{}, current_time: 0, stream_format: stream_format}}
```

Comment:
If you never return error tuple then just return a new live queue, without :ok atom. That's also how MapSet.new or Map.new work. I would do the same for other functions